src/theme/colorScheme.ts

```diff
@@ -0,0 +1,60 @@
+export enum ColorScheme {
+  Dark = "Dark",
+  Light = "Light",
+  System = "System",
+}
+
+/**
+ * Returns `true` if the value is a valid color scheme.
+ */
+export const isValidColorScheme = (value: unknown) => {
+  return Object.values(ColorScheme).includes(value as ColorScheme);
+};
+
+/**
+ * Toggles the dark color scheme according to the flag passed in.
+ *
+ * Note: Activate/deactivating the dark color scheme is a matter of adding/removing
+ *       the `.ion-palette-dark` class from the `<html>` element.
+ *
+ * Reference: https://ionicframework.com/docs/theming/dark-mode#css-class
+ *
+ * @param wantsDarkColorScheme {boolean} Whether you want the dark color scheme to be enabled.
+ */
+export const toggleDarkColorScheme = (wantsDarkColorScheme: boolean) => {
+  const htmlEl = document.documentElement;
+  htmlEl.classList.toggle("ion-palette-dark", wantsDarkColorScheme);
+};
+
+/**
+ * Checks whether the system prefers the dark color scheme.
+ */
+export const doesSystemPreferDarkColorScheme = () => {
+  const prefersDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
```

Comment:
Note: `src/components/SystemColorSchemePreferenceMonitor/SystemColorSchemePreferenceMonitor.tsx` also creates a `mediaQueryList` with the same parameter. Opportunity to consolidate?

Reply:
Consolidated.